.cirrus.yml

```diff
@@ -11,6 +11,7 @@ env:
   ARTIFACTORY_DEPLOY_PASSWORD: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-qa-deployer access_token]
   #Possible values for ARTIFACTORY_DEPLOY_REPO: sonarsource-private-qa, sonarsource-public-qa
   ARTIFACTORY_DEPLOY_REPO: sonarsource-public-qa
+  ARTIFACTORY_ACCESS_TOKEN: VAULT[development/artifactory/token/${CIRRUS_REPO_OWNER}-${CIRRUS_REPO_NAME}-private-reader access_token]
```

Comment:
Even if nothing gets deployed in private, the script `cirrus-env` injects a `settings.xml` pointing to a proper location of `.m2` folder. If `ARTIFACTORY_ACCESS_TOKEN`, settings are not injected and cleanuo_maven_cache fails because maven build didn't use the folder it expects.